local-cli/default.config.js

```diff
@@ -34,6 +34,10 @@ var config = {
 };
 
 function getRoots() {
+  var root = process.env.react_native_app_root;
```

Comment:
Actually I'm new to react-native community and I've no idea what its convention is. If the env var should be capitalised then  REACT_NATIVE_APP_ROOT would be better :)
And I just grep all existing 'process.env' references in current code base, indeed all env vars are capitalised. I agree REACT_NATIVE_APP_ROOT is better.